.rubocop_todo.yml

```diff
@@ -1589,6 +1584,30 @@ Rails/SquishedSQLHeredocs:
     - 'lib/mastodon/snowflake.rb'
     - 'lib/tasks/tests.rake'
 
+# Configuration parameters: Include.
+# Include: db/**/*.rb
+Rails/ThreeStateBooleanColumn:
```

Comment:
@ClearlyClaire did you want to deal with this rule, or just ignore it for now?

Reply:
It does not make sense to change the past migrations to satisfy these rules. It could make sense to add new migrations to do so, but I do not think this is worth it.